packages/visibility_detector/lib/src/visibility_detector_layer.dart

```diff
@@ -122,8 +122,8 @@ class VisibilityDetectorLayer extends ContainerLayer {
   /// Computes the bounds for the corresponding [VisibilityDetector] widget, in
   /// global coordinates.
   Rect _computeWidgetBounds() {
-    final r = _localRectToGlobal(this, widgetOffset & widgetSize);
-    return r.shift(paintOffset + _layerOffset);
+    final r = _localRectToGlobal(this, (paintOffset + widgetOffset & widgetSize));
```

Comment:
The parentheses around `paintOffset + widgetOffset & widgetSize` aren't necessary (and removing them would sufficiently reduce the line length to avoid formatting this across two lines when running `dart format`).

Reply:
Thank you, you are right! I have removed the parentheses now.